reference/api/settings.mdx

```diff
@@ -198,7 +198,7 @@ You can use this `taskUid` to get more details on [the status of the task](/refe
 
 ## Displayed attributes
 
-The attributes added to the `displayedAttributes` list appear in search results. `displayedAttributes` only affects the search endpoints. It has no impact on the [GET documents endpoint](/reference/api/documents#get-documents).
+The attributes added to the `displayedAttributes` list appear in search results. `displayedAttributes` only affects the search endpoints. It has no impact on the [get documents with POST](/reference/api/documents#get-documents-with-post) and [get documents with GET](/reference/api/documents#get-documents-with-get) endpoints.
```

Comment:
@maryamsulemani97 same question as above. Should we just replace the endpoint link to favor its usage or do we want to specify both endpoints?

Reply:
Same as above. Omitting the GET route might imply `displayedAttributes` impacts it.